cumulus/parachains/runtimes/assets/asset-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -80,6 +76,6 @@ impl snowbridge_pallet_system_frontend::Config for Runtime {
 	#[cfg(feature = "runtime-benchmarks")]
 	type XcmSender = DoNothingRouter;
 	type AssetTransactor = AssetTransactors;
-	type WETH = WETH;
+	type FeeAsset = FeeAsset;
```

Comment:
Do we need this?

Reply:
Yeah, it's the system-frontend pallet deployed on AH. We need to explicit the fee asset here.